xml/System.Text.Json/JsonTokenType.xml

```diff
@@ -254,8 +265,8 @@
       </ReturnValue>
       <MemberValue>9</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The token type is the JSON literal <see langword="true" />.</summary>
```

Comment:
```suggestion
        <summary>The token type is the JSON literal <see langword=<b>true</b> />.</summary>
```

Reply:
Changed your suggestion to the correct format:

```suggestion
        <summary>The token type is the JSON literal <b>true</b>.</summary>
```